bundles/FileExplorerBundle/config/pimcore/routing.yaml

```diff
@@ -0,0 +1,6 @@
+_pimcore_admin_file_explorer:
+    resource: "../../src/Controller/Admin/"
+    type:     annotation
+    prefix:   /admin/fileexplorer
```

Comment:
Add the prefix to the routes e.g.  prefix:   /admin/bundle/glossary in the [routing.yaml](https://github.com/pimcore/pimcore/blob/4d3b5dd592472b9e02fa1fec4ddb09d81b3e9afb/bundles/GlossaryBundle/config/pimcore/routing.yaml)